packages/google_maps_flutter/ios/Classes/GoogleMapController.h

```diff
@@ -9,9 +9,11 @@
 #import "GoogleMapPolygonController.h"
 #import "GoogleMapPolylineController.h"
 
+NS_ASSUME_NONNULL_BEGIN
```

Comment:
The `id _Nullable` below had opted this entire header into nullability, but it wasn't specified anywhere, which caused the analyzer warnings.